api/src/main/java/io/lumeer/api/dto/JsonOrganization.java

```diff
@@ -93,4 +93,23 @@ public String toString() {
             ", permissions=" + permissions +
             '}';
    }
+
+   @Override
+   public boolean equals(final Object o) {
+      if (this == o) {
+         return true;
+      }
+      if (o == null || getClass() != o.getClass()) {
+         return false;
+      }
+
+      final JsonOrganization that = (JsonOrganization) o;
```

Comment:
Try to generate this in IntelliJ IDEA with the following options checked:
- Accept subclasses as parameter to equals() method
- Use getters during code generation

Then replace all occurances of `JsonOrganization` in this method with `Organization` interface so it can be compared to some other implementation.